test/unittest/StatisticsBackend/InitMonitorFactoryFailsTests.cpp

```diff
@@ -41,8 +41,10 @@
 #include <fastdds_statistics_backend/types/EntityId.hpp>
 #include <fastdds_statistics_backend/types/types.hpp>
 #include <Monitor.hpp>
-#include <topic_types/typesPubSubTypes.h>
-#include <topic_types/monitorservice_typesPubSubTypes.h>
+#include <topic_types/typesPubSubTypes.hpp>
+
+#include <topic_types/monitorservice_typesPubSubTypes.hpp>
+
```

Comment:
Remove unnecessary line breaks
```suggestion
#include <topic_types/typesPubSubTypes.hpp>
#include <topic_types/monitorservice_typesPubSubTypes.hpp>
```